content/quickstart/python/tracing-beginner.md

```diff
@@ -94,9 +94,16 @@ The `main` method calls `doWork` a number of times. Each invocation also generat
 def doWork():
     # 5. Start another span. Because this is within the scope of the "main" span,
     # this will automatically be a child span.
-    with tracer.span(name="doWork"):
+    with tracer.span(name="doWork") as span:
         print("doing busy work")
-        time.sleep(0.1)
+        try:
+            time.sleep(0.1)
+        except:
```

Comment:
Please update this whole except clause to be
```python
except Exception as e:
    span.status = Status.from_exception(e)
```
which is the idiomatic way of extracting exceptions and errors with the OpenCensus-Python implementation